Xamarin.Forms.Platform.WinRT/LabelRenderer.cs

```diff
@@ -172,6 +204,7 @@ void UpdateLineBreakMode(TextBlock textBlock)
 					textBlock.TextWrapping = TextWrapping.NoWrap;
 					break;
 				case LineBreakMode.MiddleTruncation:
+					// TODO: This truncates at the end.
```

Comment:
It looks like it's not natively supported. Wanted to do some investigation. Will add to the comment.